client/components/settings/settingBody.jade

```diff
@@ -0,0 +1,72 @@
+template(name="setting")
```

Comment:
This template name should be plural, just as the directory name. `name="settings"`